content/en/build-guide/m12.adoc

```diff
@@ -23,7 +23,11 @@ image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-NWTC
 
 == Assembly Diagram == 
 
-{{< embed-pdf hidePaginator=true url="/D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+{{< embed-pdf hidePaginator=true url="./D-TET-001102-1 M12 PAPRA Controller Drawing.pdf" >}}
+
+== Circuit Diagram == 
+
+{{< embed-pdf hidePaginator=true url="./D-TET-001100-0 Papra PCBA Drawing.pdf" >}}
```

Comment:
Drawing not rendering for me ![Screenshot from 2021-05-17 08-59-16](https://user-images.githubusercontent.com/14095576/118519796-491c5f80-b6ee-11eb-8a78-a50eb01ceb5d.png)

Reply:
Should not have a second pdf render on the same page, the embedding can't handle it.

Reply:
should be fixed now

Reply:
fix looks good now